test/combo-test.rkt

```diff
@@ -0,0 +1,73 @@
+#lang racket
+
+(require (for-syntax rackunit
+                     syntax/parse)
+         rackunit
+         "helpers.rkt"
+         "game-schema-test.rkt"
+         "../lib/mock-device.rkt"
+         "../lib/schemas.rkt"
+         "../lib/yomi-lib.rkt")
+
+(define-syntax check-def-failure
+  (syntax-parser
+    [(_ def msg)
+     (check-exn (regexp (syntax->datum #'msg))
+                (lambda () (local-expand #'def 'module-begin null)))
+     #'(void)]))
+
+(using-character "character-schema-test.rkt")
+
+(define-combo basic-link HP & HP)
+
+(define-combo basic-cancel HP ~ HP)
+
+(define-combo link-then-cancel HP & HP ~ HP)
+
+(define-combo cancel-then-link HP ~ HP & HP)
```

Comment:
Could you please add a test or two that uses more advanced moves? Something with directions and multiple button presses? Also, could you add a comment for the tests below noting that these need to be updated when the library is modified to better support holds?